packages/oauth-client/README.md

```diff
@@ -0,0 +1,259 @@
+# @cumulus/oauth-client
+
+Utilities for OAuth authentication using
+[NASA Earthdata Login](https://urs.earthdata.nasa.gov/) and [AWS Cognito](https://aws.amazon.com/cognito/).
+
+## Versioning
+
+Cumulus uses a modified semantic versioning scheme and minor releases likely
+include breaking changes.
+
+Before upgrade, please read the Cumulus
+[release notes](https://github.com/nasa/cumulus/releases) before upgraded.
+
+It is strongly recommended you do not use `^` in your `package.json` to
+automatically update to new minor versions. Instead, pin the version or use `~`
+to automatically update to new patch versions.
+
+## Installation
+
+```bash
+$ npm install @cumulus/oauth-client
+```
+
+## Earthdata Login Usage Example
+
+```js
+const { EarthdataLoginClient } = require('@cumulus/oauth-client');
+
+const client = new EarthdataLogin({
+  clientId: 'my-client-id',
+  clientPassword: 'my-client-password',
+  earthdataLoginUrl: 'https://earthdata.login.nasa.gov',
+  redirectUri: 'http://my-api.com'
+});
+```
+## Cognito Usage Example
+
+```js
+const { CognitoClient } = require('@cumulus/oauth-client');
+
+const client = new CognitoClient({
+  clientId: 'my-client-id',
+  clientPassword: 'my-client-password',
+  cognitoLoginUrl: 'https://earthdata.login.nasa.gov',
```

Comment:
Good catch. I'm using the SIT URL here. I'm not sure what the prod URL will end up being but as an example this might be fine?

https://github.com/nasa/cumulus/pull/2209/commits/b9e78a517a7e1328aa133e7cb916023196469f79